Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/CreatePrismaCloudComputeComplianceReportButton.yml

```diff
@@ -0,0 +1,28 @@
+args:
+- isArray: true
+  name: table
+  required: true
+  description: The table of data.
```

Comment:
Is this a comma-separated list?   I see that isArray is true.

Reply:
no this needs to be a `list` of objects. the automation loops through it to create a table.